src/components/DownloadSalesEvidenceListButton.tsx

```diff
@@ -17,10 +17,11 @@ export default function DownloadSalesEvidenceListButton (props: DownloadButtonPr
         const dateSold = propertyObject.Sale_Date
         const salePrice = propertyObject.Sale_Price
 
-        let csvRow = `"${propertyAddress}","${landArea}","${buildArea}","${dateSold}","${salePrice}"\r\n`
-        csvRow = csvRow.replace(/null/g, '-')
-
-        return csvRow
+        if (landArea || buildArea || dateSold || salePrice) {
+            let csvRow = `"${propertyAddress}","${landArea}","${buildArea}","${dateSold}","${salePrice}"\r\n`
+            csvRow = csvRow.replace(/null/g, '-')
+            return csvRow
+        }
```

Comment:
csvRow will be undefined in the else clause. I'd suggest moving 

```
csvRow = csvRow.replace(/null/g, '-')
        return csvRow	            return csvRow
``` 
outside the if

Reply:
What do you mean? There is no else clause.

Reply:
the implicit else clause.

```
if (landArea || buildArea || dateSold || salePrice) {
            csvRow = `"${propertyAddress}","${landArea}","${buildArea}","${dateSold}","${salePrice}"\r\n`
            csvRow = csvRow.replace(/null/g, '-')
} else { 
// csvRow is undefined
}
```

Reply:
You didn't make any changes. I suggest you move
csvRow = csvRow.replace(/null/g, '-')

outside the if